Majasdarbs/03-majasdarbs/Portfolio/styles.css

```diff
@@ -0,0 +1,203 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@200;300&display=swap');
+
+/* Global styles */
+
+*{
+    
+    margin:0;
+    padding:0;
+    box-sizing: border-box;
+    line-height: 2em;
+}
+
+body {
```

Comment:
Consider breaking css to smaller chunks of code. theme.css, normalise.css, responsive.css etc.